core/src/main/java/hudson/model/AbstractCIBase.java

```diff
@@ -88,12 +90,12 @@ protected void killComputer(Computer c) {
         c.kill();
     }
 
-    /* =================================================================================================================
-    * Package-protected, but accessed API
-    * ============================================================================================================== */
-
-    /*package*/ final CopyOnWriteArraySet<String> disabledAdministrativeMonitors = new CopyOnWriteArraySet<String>();
+	final CopyOnWriteArraySet<String> disabledAdministrativeMonitors = new CopyOnWriteArraySet<String>();
```

Comment:
Not a fan of `<tab>` indentation

Reply:
Oh Sorry. I didn't knew that space was there instead of tab !!